plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-shipping-method-block/style.scss

```diff
@@ -1,4 +1,4 @@
-.wc-block-checkout__shipping-method-container {
+div.wc-block-checkout__shipping-method-container {
```

Comment:
I'm mildly concerned about targeting element types to increase specificity because I could imagine this breaking again in future without knowing it, but I think the risk is low for this one change.

I guess one other option could be to use class stacking, but that also feels like a bit of  hack, just more resilient to changes in html structure (e.g. 

`.wc-block-checkout__shipping-method-container.wc-block-checkout__shipping-method-container {`

But I also can see it's a bit of a hack too so I'm just mentioning this rather than asking for a change.

Reply:
Ok, good point, class names are still not guaranteed not to change, but it's probably slightly more robust than the element. Cheers for the suggestion!